src/scripts/installOED.sh

```diff
@@ -160,7 +160,7 @@ else
 			fi
 		else
 			# There was no createdb error so assume database ready for use so stop process
-			printf "%s\n" "User creation had no errors so default user 'test@example.com' with password 'password' should exist"
+			printf "%s\n" "User creation had no errors so default user 'testUser' with password 'password' should exist"
```

Comment:
This outputs testUser and not the actual name. I think it would be good to create a variable at the top of the script and use it here and in the npm command above.

Reply:
Will do.